### PR TITLE
updated get_run_details.sl and get_workspace_details.sl and yaml files

### PR DESCRIPTION
**Pull request by ameydwivedi**
updated get_run_details.sl and get_workspace_details.sl and yaml files for same.

Please review.

### CODE REVIEW

File: content/io/cloudslang/hashicorp/terraform/runs/get_run_details.sl [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+#! @input response_character_set: The character encoding to be used for the HTTP response. If responseCharacterSet is
+#!                                empty, the charset from the 'Content-Type' HTTP response header will be used. If
+#!                                responseCharacterSet is empty and the charset from the HTTP response Content-Type
+#!                                header is empty, the default value will be used. You should not use this for
+#!                                method=HEAD or OPTIONS.Default: UTF-8
+#!                                Optional
+#!
+#! @output return_result: The percent-decoded 'url'. In case of an error this output will contain the error message.
+#! @output status_code: The HTTP status code for Terraform API request.
+#!
```
Comment (ericandreica) [inline content/io/cloudslang/hashicorp/terraform/runs/get_run_details.sl]: It's a best practice to have an exception output.

File: content/io/cloudslang/hashicorp/terraform/runs/get_run_details.sl [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+#! @input connections_max_total: The maximum limit of connections in total.Default: 20
+#!                               Optional
+#! @input response_character_set: The character encoding to be used for the HTTP response. If responseCharacterSet is
+#!                                empty, the charset from the 'Content-Type' HTTP response header will be used. If
+#!                                responseCharacterSet is empty and the charset from the HTTP response Content-Type
+#!                                header is empty, the default value will be used. You should not use this for
+#!                                method=HEAD or OPTIONS.Default: UTF-8
+#!                                Optional
+#!
+#! @output return_result: The percent-decoded 'url'. In case of an error this output will contain the error message.
```
Comment (ericandreica) [inline content/io/cloudslang/hashicorp/terraform/runs/get_run_details.sl]: The description is a little ambiguous. Decoded url sound better.

File: content/io/cloudslang/hashicorp/terraform/runs/get_run_details.sl (hunk 1 above)
Comment (ameydwivedi) [reply content/io/cloudslang/hashicorp/terraform/runs/get_run_details.sl]: resolved

File: content/io/cloudslang/hashicorp/terraform/runs/get_run_details.sl (hunk 2 above)
Comment (ameydwivedi) [reply content/io/cloudslang/hashicorp/terraform/runs/get_run_details.sl]: resolved

**Comment by ameydwivedi** [comment]
@ericandreica , i have updated file as per your comments. Please approve if no more changes needed.

File: content/io/cloudslang/hashicorp/terraform/runs/get_run_details.sl (hunk 1 above)
Comment (ericandreica) [reply content/io/cloudslang/hashicorp/terraform/runs/get_run_details.sl]: "while apply run" is ambigouos.

File: content/io/cloudslang/hashicorp/terraform/runs/get_run_details.sl [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    
+  java_action: 
+    gav: 'io.cloudslang.content:cs-hashicorp-terraform:1.0.0-RC4'
+    class_name: 'io.cloudslang.content.hashicorp.terraform.actions.runs.GetRunDetails'
+    method_name: 'execute'
+  
+  outputs: 
+    - return_result: ${get('returnResult', '')} 
+    - status_code: ${get('statusCode', '')} 
+
```
Comment (ericandreica) [inline content/io/cloudslang/hashicorp/terraform/runs/get_run_details.sl]: here should be the exception output

Comment (ameydwivedi) [reply content/io/cloudslang/hashicorp/terraform/runs/get_run_details.sl]: resolved.

File: content/io/cloudslang/hashicorp/terraform/runs/get_run_details.sl (hunk 1 above)
Comment (ameydwivedi) [reply content/io/cloudslang/hashicorp/terraform/runs/get_run_details.sl]: resolved

**Pull request merged**